xija/gui_fit/plots.py

```diff
@@ -161,27 +161,17 @@ def clearLayout(layout):
 
 
 def annotate_limits(limits, ax, dir='h'):
+    from xija.limits import get_limit_color
     if len(limits) == 0:
         return []
     lines = []
     draw_line = getattr(ax, f'ax{dir}line')
-    opts = [
-        ('planning.data_quality.high.acisi', '-.', 'blue'),
-        ('planning.data_quality.high.aciss', '-.', 'purple'),
-        ('planning.penalty.high', '-.', 'gray'),
-        ('planning.warning.low', '-', 'green'),
-        ('planning.warning.high', '-', 'green'),
-        ('odb.caution.low', '-', 'gold'),
-        ('odb.caution.high', '-', 'gold'),
-        ('odb.warning.low', '-', 'red'),
-        ('odb.warning.high', '-', 'red'),
-        ('planning.zero_feps.low', '--', 'dodgerblue')
-    ]
-    for (limit_name, ls, color) in opts:
-        if limit_name in limits:
-            lines.append(
-                draw_line(limits[limit_name], ls=ls, color=color)
-            )
+    for limit in limits:
+        if limit == "unit":
+            continue
+        lines.append(
+            draw_line(limits[limit], color=get_limit_color(limit))
```

Comment:
I think this should pay attention to the limit units (indeed many of them are not degC).

Reply:
@taldcroft what do you want it to do exactly? This loop is to draw the lines and since the `"unit"` item is in the same dict as the limit definitions, it needs to skip over it which is the reason for lines 170-171.

Reply:
E.g the limits for the PFTANK2T model are in degF, but the models are always computed and calculated in degC. Maybe I'm missing the code, but somewhere along the way the limit values need to be converted degC.